packages/link/src/utils/wagmi.ts

```diff
@@ -0,0 +1,54 @@
+import { createConfig, http } from 'wagmi'
+import {
+  bsc,
+  mainnet,
+  polygon,
+  avalanche,
+  arbitrum,
+  optimism
+} from 'wagmi/chains'
+import { walletConnect, coinbaseWallet, injected } from 'wagmi/connectors'
+
+export const config = createConfig({
+  chains: [mainnet, bsc, polygon, avalanche, arbitrum, optimism],
```

Comment:
Let's ask Jason, what are the planned EVM chains, so we can include them as well, so we don't need a new SDK release when adding those